frontend/src/app/my-courses/course/office-hours/office-hours-get-help/office-hours-get-help.component.ts

```diff
@@ -161,8 +161,8 @@ export class OfficeHoursGetHelpComponent implements OnInit, OnDestroy {
       next: (_) => {
         this.pollData();
       },
-      error: (_) => {
-        this.snackBar.open(`Could not create a ticket at this time.`, '', {
+      error: (err) => {
+        this.snackBar.open(`${err.error.message}`, '', {
```

Comment:
**issue:** It might be better if ticket creation is completely disabled for the user _before_ they even attempt to create it and send a request to the backend. Both to reduce the number of unnecessary requests and to avoid situations where a user unknowingly creates a ticket before the cooldown/limit is over, writes a whole description, and then loses it. This will obviously require more work and refactoring though, so could be relegated to a future ticket.